tests/cypress/integration/2-general-onboarding-flow/site-features.cy.js

```diff
@@ -36,19 +37,18 @@ describe( 'Site Features', function () {
 		);
 	} );
 
-	if(GetPluginId()=='bluehost'){
-	it( 'Check to make sure sidebar opens, content is in place and close sidebar', () => {
-		CheckIntroPanel( '__site-features', 'Features' );
-		CheckIllustrationPanel();
-		CheckInfoPanel();
-		CheckHelpPanelLinks();
-	} );
-	}
-	else{
+	if ( GetPluginId() == 'bluehost' ) {
+		it( 'Check to make sure sidebar opens, content is in place and close sidebar', () => {
+			CheckIntroPanel( '__site-features', 'Features' );
+			CheckIllustrationPanel();
+			CheckInfoPanel();
+			CheckHelpPanelLinks();
+		} );
+	} else {
 		it( 'Check to make sure Sidebar opens', () => {
 			BasicSidebarCheck();
 		} );
-	};
+	}
```

Comment:
Why have we removed ';' at the end?

Reply:
Its due to linting. I used command provided earlier.
`npx wp-scripts lint-js <file-path>  --no-ignore --fix`